inst/shiny-server/server/FCESummary.R

```diff
@@ -25,7 +25,7 @@ output$FCESummary.Overview.Download <- downloadHandler(
   },
   content = function(file) {
     df <- FCE_runtime_summary_condensed()
-    df <- df[input[["FCE_SAMPLE_rows_all"]]]
+    df <- df[input[["table_FV_overview_rows_all"]]]
```

Comment:
WE should probably discuss the naming scheme. Let's do it when Richard joins us

Reply:
Yes, that is indeed something we should improve in a later version